core/templates/list.h

```diff
@@ -415,6 +415,19 @@ class List {
 	/**
 	 * find an element in the list,
 	 */
+	template <typename T_v>
+	const Element *find(const T_v &p_val) const {
```

Comment:
I added a `const` version of `List::find` to support using `find` in `const` contexts.